sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -424,3 +424,26 @@ function buildRawAuthorizationRule(authorizationRule: AuthorizationRule): any {
   };
   return rawAuthorizationRule;
 }
+
+/**
+ * @ignore
+ * Helper utility to extract the relative path from given URL,
+ * or undefined / empty if not existing.
+ * Valid relative paths used in the ATOM based management API are in format
+ *     - `<queue-name>`
+ *     - `<topic-name>`
+ *     - `<topic-name>/Subscriptions/<subscription-name>`
+ *     - `<topic-name>/Subscriptions/<subscription-name>/Rules/<rule-name>`
+ * @param url
+ */
+export function getRelativePathOrUndefined(url: string): string | undefined {
+  let stringValue = getStringOrUndefined(url);
+  if (stringValue == undefined) {
+    return undefined;
+  }
+
+  // Skip the sb:// prefix in URL
+  stringValue = stringValue.substring(5);
```

Comment:
Are we guaranteed that the string will have the prefix? If the prefix is not there, this will end up removing the start of the string.

Reply:
This was because we accept input as just the relative path and we are in control of the URL being constructed and is done so to specifically begin with "sb://"

But they can technically begin with http:// or https:// as well - updated code to factor these in.

The forwardTo and forwardDeadletterMessagesTo fields are ensured to always be valid URIs. 
.NET SDK seems to return the full URI when it comes to response though it accepts relative path as the input. (Thus using this as fallback behavior in JS if in event of unlikely possibility that a new/unknown network protocol is used in the FQDN through some other SDK or portal)

Similar discussion as in https://github.com/Azure/azure-sdk-for-js/pull/6382#discussion_r358398026
For a better user experience, symmetry in values being set and retrieved would be preferable I think. Also important, since we use the response information to construct requests in some API.

Reply:
Resolving this comment thread as the code in question is no longer in use in the recent iterations of the PR